OxfordBodleian/Juel-Jensen/BDLaethc14.xml

```diff
@@ -207,76 +664,75 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <q xml:lang="gez">ዘቀትር፡</q> (<locus target="#127rb"/>),
+                                        <q xml:lang="gez">ዘሰዓት፡</q> (<locus target="#127vb"/>),
+                                        <q xml:lang="gez">ዘሰርክ፡</q> (<locus target="#128ra"/>),
+                                        <q xml:lang="gez">ዘእሁድ፡ ሰንበት፡</q> (<locus target="#128rb"/>).
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
                     </physDesc>
-                    
-                    <additional>
-                        <adminInfo>
-                            <recordHist>
-                                <source>
-                                    <!--  <listBibl type="secondary">
-                                        <bibl></bibl>
-                                    </listBibl>-->
-                                </source>
-                            </recordHist>
-                        </adminInfo>
-                    </additional>
-                    
-                <!--    <history>
-                        <origin>
-                            <origDate when="" evidence=""/>
-                        </origin>
-                        <provenance>  </provenance>
-                    </history>-->
-                    
-                    <msPart xml:id="p1">
-                        <msIdentifier>
-                        </msIdentifier>
-                        
-                        <msContents>
-                            <summary/>
-                            
-                       <!--     <msItem xml:id="p1_i1">
-                                <locus from="2r" to=""/>
-                                <title type="" ref=""/>
-                               <textLang xml:lang="gez"/>
-                            </msItem>-->
-                        </msContents>
-                        
-                        
-                    </msPart>
-                    
-                    <msPart xml:id="p2">
-                        <msIdentifier>
-                        </msIdentifier>
-                        
-                        <msContents>
-                            <summary/>
-                          <!--  <msItem xml:id="p2_i1">
-                                <locus from="" to=""/>
-                                <title type="" ref=""/>
-                                <textLang xml:lang="gez"/>
-                            </msItem>-->
-                        </msContents>
+                   
+                        <history>
+                            <origin>
+                                <origDate from="1650" to="1800" evidence="lettering"/>
+                            </origin>
+                            <provenance>According to <ref target="#coloph1"/>, the manuscript belonged to <placeName ref="LOC7349DabraMi">Dabra Mikāʾel</placeName>
```

Comment:
If you consider the colophon to refer to the entire circulation unit as it is now, I would encode this in the physDesc pertaining to the entire manuscript, and not twice in the msParts.

Reply:
The colophon is written in the same hand as that of msPart2. It also provides the name of the scribe of msPart2. So, I thought it was appropriate to encode it in the msPart2 physDesc. Now, I am a bit confused as to the best way to encode the history of this composite Ms, because, as paleography shows, the two msParts clearly have different ages, so I would be inclined to encode each dating in their respective msPart.
Is the following encoding (just after the physDesc pertaining to the entire manuscript) a good solution in order not to multiply the sections and scatter the pieces of information?
 ```
<history>
                         <origin>
                            <origDate from="1500" to="1600" evidence="lettering" corresp="#p1"/>
                            <origDate from="1650" to="1800" evidence="lettering" corresp="#p2"/>
                        </origin>
                        <provenance>According to <ref target="#coloph1"/>, the manuscript belonged to <placeName ref="LOC7349DabraMi">Dabra Mikāʾel</placeName>
                            of the city of <placeName ref="LOC6450Zatta">Zāttā</placeName> in <placeName ref="LOC6573NorthWallo">Northern Wallo</placeName>.
                            The same church is also mentioned in <ref target="#a2"/>.
                        </provenance>
                    </history>
```